src/modules/firewall/src/lib/Firewall.h

```diff
@@ -19,169 +24,135 @@ class FirewallLog
 public:
     static OSCONFIG_LOG_HANDLE Get()
     {
-        return m_logFirewall;
+        return LOG_HANDLE;
     }
     static void OpenLog()
     {
-        m_logFirewall = ::OpenLog(FIREWALL_LOGFILE, FIREWALL_ROLLEDLOGFILE);
+        LOG_HANDLE = ::OpenLog(FIREWALL_LOGFILE, FIREWALL_ROLLEDLOGFILE);
     }
     static void CloseLog()
     {
-        ::CloseLog(&m_logFirewall);
+        ::CloseLog(&LOG_HANDLE);
     }
 
 private:
-    static OSCONFIG_LOG_HANDLE m_logFirewall;
+    static OSCONFIG_LOG_HANDLE LOG_HANDLE;
```

Comment:
>LOG_HANDLE [](http://example.com/codeflow?start=31&length=10)

Previous name is better. Variables like this need to use correct casing. They are not C macros.